src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessTokenClient/Services/AccessTokenGenerator.cs

```diff
@@ -93,7 +102,12 @@ private string GenerateAccessToken(string issuer, string app, SigningCredentials
                 };
 
                 SecurityToken token = tokenHandler.CreateToken(tokenDescriptor);
-                string tokenstring = tokenHandler.WriteToken(token);
+                tokenstring = tokenHandler.WriteToken(token);
+
+                var cacheEntryOptions = new MemoryCacheEntryOptions()
```

Comment:
the cacheEntryOptions could be aread only property you declare in the constructor so a new one isn't created for each entry.

Reply:
It isn't significant for performance. It's only used in one method, so I think it's cleaner to keep it local to the method.